process_onestop_sr_report/preprocessing.py

```diff
@@ -273,6 +271,20 @@ def _load_raw_data(data_path) -> pd.DataFrame:
 
 
 def preprocess_data(args: ArgsParser) -> pd.DataFrame:
+    """Preprocesses onestopgaze raw reports
+
+    NOTE: To extract surprisal from state-spaces/mamba-* variants, better to first run:
+    >>> pip install causal-conv1d
+    >>> pip install mamba-ssm
+    """
+    # If this won't be here the Surprisal columns won't be added to the base columns if they are not in the base columns in the first place
```

Comment:
First that is weird because I am pretty sure it worked.
Second, the process_args function is a more suitable spot if necessary IMO